browser/ui/views/frame/brave_tab_strip_region_container.cc

```diff
@@ -0,0 +1,284 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/views/frame/brave_tab_strip_region_container.h"
+
+#include <memory>
+#include <utility>
+
+#include "brave/browser/ui/views/sidebar/sidebar_button_view.h"
+#include "brave/browser/ui/views/tabs/brave_new_tab_button.h"
+#include "brave/browser/ui/views/tabs/brave_vertical_tab_utils.h"
+#include "chrome/browser/themes/theme_properties.h"
+#include "chrome/browser/ui/color/chrome_color_id.h"
+#include "chrome/browser/ui/tabs/tab_style.h"
+#include "chrome/browser/ui/views/tabs/tab_search_button.h"
+#include "chrome/grit/generated_resources.h"
+#include "ui/base/l10n/l10n_util.h"
+#include "ui/base/metadata/metadata_impl_macros.h"
+#include "ui/base/theme_provider.h"
+#include "ui/gfx/canvas.h"
+#include "ui/views/accessibility/accessibility_paint_checks.h"
+#include "ui/views/controls/scroll_view.h"
+#include "ui/views/layout/fill_layout.h"
+#include "ui/views/layout/flex_layout.h"
+
+namespace {
+
+class ScrollHeaderView : public views::Button {
+ public:
+  METADATA_HEADER(ScrollHeaderView);
+
+  ScrollHeaderView(Button::PressedCallback callback, TabStrip* tab_strip)
+      : Button(std::move(callback)), tab_strip_(tab_strip) {
+    // TODO(sangwoo.ko) Temporary workaround before we have a proper tooltip
```

Comment:
please create an a tracking issue for this

Reply:
done.